pylint/lint.py

```diff
@@ -748,6 +748,11 @@ def error_mode(self):
     def python3_porting_mode(self):
         """Disable all other checkers and enable Python 3 warnings."""
         self.disable("all")
+        # re-enable some errors, or 'print', 'raise', 'async', 'await' will mistakenly lint fine
```

Comment:
Can you use the symbolic names here instead? e.g. `fatal`, `syntax-error` and so on.

Reply:
done